lib/rides/pages/search_suggestion_page.dart

```diff
@@ -309,19 +348,275 @@ class _SearchSuggestionPage extends State<SearchSuggestionPage> {
+                () => _minHospitalityRating = newRating.toInt(),
+              ),
+            ),
+          ],
+          TextButton(
+            onPressed: () => innerSetState(() => _isRatingExpanded = !_isRatingExpanded),
+            child: Text(_isRatingExpanded ? S.of(context).retract : S.of(context).expand),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildFeaturesFilter(void Function(void Function()) innerSetState) {
+    List<Feature> shownFeatures = _isFeatureListExpanded ? Feature.values : _commonFeatures;
+    return filterCategory(
+      S.of(context).pageSearchSuggestionsFilterFeatures,
+      Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Wrap(
+            runSpacing: -10,
+            children: List.generate(
+              shownFeatures.length,
+              (index) {
+                Feature feature = shownFeatures[index];
+                bool featureSelected = _selectedFeatures.contains(feature);
+                return FilterChip(
+                  avatar: feature.getIcon(context),
+                  label: Text(feature.getDescription(context)),
+                  selected: _selectedFeatures.contains(feature),
+                  tooltip: featureSelected
+                      ? S.of(context).pageSearchSuggestionsFilterFeaturesDeselectTooltip
+                      : S.of(context).pageSearchSuggestionsFilterFeaturesSelectTooltip,
+                  onSelected: (selected) {
+                    if (featureSelected) {
+                      innerSetState(() => _selectedFeatures.remove(feature));
+                    } else {
+                      Feature? mutuallyExclusiveFeature = _selectedFeatures
+                          .firstWhereOrNull((selectedFeature) => selectedFeature.isMutuallyExclusive(feature));
+                      if (mutuallyExclusiveFeature != null) {
+                        String description = mutuallyExclusiveFeature.getDescription(context);
+                        String text = S.of(context).pageProfileEditProfileFeaturesMutuallyExclusive(description);
+                        SemanticsService.announce(text, TextDirection.ltr);
+                        ScaffoldMessenger.of(context).showSnackBar(
+                          SnackBar(content: Text(text)),
+                        );
+                        return;
+                      }
+                      innerSetState(() => _selectedFeatures.add(feature));
+                    }
+                  },
+                );
+              },
+            ),
+          ),
+          TextButton(
+            onPressed: () => innerSetState(() => _isFeatureListExpanded = !_isFeatureListExpanded),
+            child: Text(_isFeatureListExpanded ? S.of(context).retract : S.of(context).expand),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildDeviationFilter() {
+    return filterCategory(
+      S.of(context).pageSearchSuggestionsFilterDeviation,
+      Row(
```

Comment:
should also include the ability to just select "whole day"

Reply:
Food for thought: Shouldn't that be an option when looking for a ride? Instead of entering a time, you could check a checkbox "whole day" and then the filter option becomes the max deviation of days

Reply:
Sure we can do it that way too... Is there space for that?

Reply:
Btw if we do that, we can think about removing the time frame that the user can choose? I know that Blablacar always shows the whole day, and then you can go left and right to view other dates.

Reply:
I think we should discuss this in a meeting and set it aside for another user story. I believe we discussed with AG that we should include the time in the search and the algorithm might play a big part in that (is there even an option for the whole day in the algorithm? Or will we have to fire multiple requests to get the results if it only outputs rides with a certain time deviation?) I would personally prefer giving users the option to choose the time (since some trips might be time-critical) but maybe making whole day the default